bokeh/charts/chart.py

```diff
@@ -82,14 +82,14 @@ class Chart(Plot):
     def __init__(self):
 
         # Initializes then gets default properties
-        super(Chart, self).__init__(id=self._options.id or make_id())
+        # NOTE: we use option._id here since ChartOptions converts id arg to _id
+        super(Chart, self).__init__(id=self._options._id or make_id())
```

Comment:
hang on - why are we using the id from the options object? (in fact, why is the options object a Model, does it appear in the Document? If not it could/should be a HasProps which has no id)

Why not omit this id stuff entirely and leave the id autogenerated the way it is for every other model?